infrastructure/main.tf

```diff
@@ -20,7 +39,7 @@ module "key_vault" {
 
 module "aks" {
   source              = "./modules/resource_templates/aks"
-  aks_name            = var.aks_name
+  aks_name            = local.aks_name
```

Comment:
Just as a minor idea, just missed the review point of the predecessor PR (https://github.com/buerokratt/Infrastructure/pull/13) but I noticed within each resource module e.g. AKS, variables are prefixed with their actual resource name e.g. in modules/resourcetemplates/aks/variables.tf the variable which is used as the name of the cluster is called 'aks_name', could we just call that variable 'name' instead since it is already encapsulated in a 'aks' folder and resource, just thought it might be a neater naming convention, thoughts?

Reply:
Very good point 👍🏽 I've implemented your suggestion
Thanks! 😃